packages/extension-chakra-storefront/src/components/product-view-modal/bundle.jsx

```diff
@@ -51,7 +51,7 @@ const BundleProductViewModal = ({product: bundle, isOpen, onClose, updateCart, .
         {parameters: {ids: childProductIds, allImages: true}},
         {
             enabled: Boolean(childProductIds),
-            keepPreviousData: true
+            placeholderData: (previousData) => previousData
```

Comment:
Did you decided to inline a function instead of using keepPreviousData for any specific reason?

Reply:
I was under the impression that `@tanstack/react-query` was not a direct dependency of the extension, but i just checked and apparently I was wrong...

I'll import `keepPreviousData`